constraints/src/main/java/gov/nasa/jpl/aerie/constraints/time/Spans.java

```diff
@@ -157,6 +160,16 @@ public Spans split(final Interval bounds, final int numberOfSubSpans, final Incl
     });
   }
 
+  public boolean isSubsetOf(final Spans otherSpans){
+    return new HashSet<>(otherSpans.intervals).containsAll(this.intervals);
+  }
```

Comment:
Actually, in that precise case, I believe this is the behavior I want. The spans for coexistence can only either be the original ones or disappear one by one strictly (without leaving any subinterval) as new activities are anchored to each of them.

Reply:
Sorry for the late response. I'm OK with this behavior then, but I think it should have a different name and a doc comment because it could be misleading. Maybe `.isCollectionSubsetOf`? I'm not sure. I'm happy as long as its a weird enough name to force people to read the doc comment.

Reply:
Done.